README.md

```diff
@@ -88,7 +88,12 @@ Usage of hetty:
 
 **Note:** There is no built-in in support yet for generating a CA certificate.
 This will be added really soon in an upcoming release. In the meantime, please
-use `openssl` (_TODO: add instructions_).
+use [openssl](https://www.openssl.org/docs/man1.0.2/man1/openssl.html).
```

Comment:
As of https://github.com/dstotijn/hetty/commit/8b047478557755757fe7ced0d36e78b25c295dea, this shouldn't be needed anymore 🙌

Reply:
yeah that wasn't supposed to be in here anyways but in a separate PR